build.gradle

```diff
@@ -14,5 +15,51 @@ allprojects {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.2.1'
-}
\ No newline at end of file
+    gradleVersion = '2.4'
+}
+
+task updateDemos() << {
```

Comment:
I would leave the script or we can move it buildSrc. Running it from time
to time makes sure that everything is as we want it.
Am 26.06.2015 17:52 schrieb "Paul Blundell" <notifications@github.com>:

> In build.gradle
> <https://github.com/novoda/android-demos/pull/8#discussion_r33368344>:
>
> > \ No newline at end of file
> > +    gradleVersion = '2.4'
> > +}
> > +
> > +task updateDemos() << {
>
> can this be deleted now? cool script tho
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/android-demos/pull/8/files#r33368344>.
>